internal/network/command_create.go

```diff
@@ -122,14 +129,17 @@ func (c *command) create(cmd *cobra.Command, args []string) error {
 
 	createNetwork := networkingv1.NetworkingV1Network{
 		Spec: &networkingv1.NetworkingV1NetworkSpec{
-			DisplayName:     networkingv1.PtrString(name),
 			Cloud:           networkingv1.PtrString(cloud),
 			Region:          networkingv1.PtrString(region),
 			ConnectionTypes: &networkingv1.NetworkingV1ConnectionTypes{Items: connectionTypes},
 			Environment:     &networkingv1.ObjectReference{Id: environmentId},
 		},
 	}
 
+	if name != "" {
+		createNetwork.Spec.SetDisplayName(name)
```

Comment:
It will be an empty string. 

The `display_name` field is optional in the [Networking API spec](https://docs.confluent.io/cloud/current/api.html#tag/Networks-(networkingv1)/operation/createNetworkingV1Network) for all of the `create` endpoints. 

I will also let @kabraham13 to chime in here.

Reply:
We should match what is required in the API spec.

Now why its not a required field in the API spec I am not sure. Predates my time